site/public/js/system-update.js

```diff
@@ -140,6 +140,7 @@ function toggleAllReleases(toggleAllButton) {
     //  if toggleAllButton has collapsed - will collapse all
     //  if toggleAllButton doesn't have collapsed - will expand all
     $('.btn-toggle-release').each( (index, button) => {
+        // eslint-disable-next-line eqeqeq
         if ($(toggleAllButton).hasClass('collapsed') != $(button).hasClass('collapsed')) {
```

Comment:
```suggestion
        if ($(toggleAllButton).hasClass('collapsed') !== $(button).hasClass('collapsed')) {
```

[`$.hasClass`](https://api.jquery.com/hasclass/) only returns a boolean.